docs/threat_models/treasury/treasury.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
+header Agoric Operating Systems
+footer v0.0.1
+title Treasury
+
+skinparam componentStyle rectangle
+skinparam defaultTextAlignment center
+
+
+actor "TreasuryCreator aka Governance" as tc
+
+actor Borrower
+
+node AdjustBalancesInvitation
+node "CloseVaultInvitation"
+node LiquidationPromise
+node LiquidationSeat {
+    circle getPayoff
+    circle getPayoffs
```

Comment:
Is there a reason to include both of these?

Reply:
Both of these methods appear in the Treasury contract, and should be represented independently.  (If they aren't in use, it would be worth considering eliminating one for the sake of simplicity.)

Reply:
Oops I got my wires crossed, `s/Payoff/Payout/g`.  These both appear to be callable in the contract.